website/docs/docs/deploy/state-aware-setup.md

```diff
@@ -163,16 +166,56 @@ models:
           period: hour
           updates_on: all
 ```
+</File>
 
+</TabItem>
+<TabItem value="yml" label="Project file">
+
+<File name="dbt_project.yml">
+  
+```yaml
+models:
+  [<resource-path>](/reference/resource-configs/resource-path):
+    [+](/reference/resource-configs/plus-prefix)[freshness](/reference/resource-properties/freshness):
+      build_after: 
+        count: 4
+        period: hour
+        updates_on: all 
+```
 </File>
 
+</TabItem>
+<TabItem value="sql" label="Config block">
+
+<File name="models/<filename>.sql">
+  
+```sql
```

Comment:
Looks like this might be missing some formatting 
- A bracket missing after `build_after`
- Commas after count, period, update_on
- Indentations

```
{{
    config(
        freshness={
            "build_after": {
                "count": 4,
                "period": "hour",
                "updates_on": "all"
            }
        }
    )
}}
```